app/client/views/residencies/forms/edit/edit.html

```diff
@@ -7,53 +7,79 @@
           <h4 class="modal-title">
             {{_ "editResidencyModal-header" }}
           </h4>
-        </div> <!-- / .modal-header -->
+        </div>
+        <!-- / .modal-header -->
         {{# if Template.subscriptionsReady }}
         <div class="modal-body">
           <!-- check if residency exists - editing -->
           {{# if residency }}
-            {{#autoForm id="editResidencyForm" type="update" collection="Residencies" doc=residency }}
-
-              <label for="residentId">
-                {{_ "residencies.residentId.label" }}
-              </label>
-              {{> afFieldInput name="residentId" options=residentIdOptions }}
-
-              <label for="homeId">
-                {{_ "residencies.homeId.label" }}
-              </label>
-              {{> afFieldInput name="homeId" options=homeSelectOptionsWithGroups }}
-
-              <label for="moveIn">
-                {{_ "residencies.moveIn.label" }}
-              </label>
-              {{> afFieldInput name="moveIn" class="dateField" buttonClasses="fa fa-calendar" }}
-
-              <label for="moveOut">
-                {{_ "residencies.moveOut.label" }}
-              </label>
-              {{> afFieldInput name="moveOut" class="dateField" buttonClasses="fa fa-calendar" }}
-
-              <br/>
-
-              <button type="submit" class="btn btn-success">
-                {{_ "editResidencyForm-save" }}
-              </button>
-              <button type="button" class="btn btn-danger" data-dismiss="modal">
-                {{_ "editResidencyForm-cancel" }}
-              </button>
-
-            {{/ autoForm }}
+          {{#unless addNewResidencyClicked}}
+          {{#unless residency.moveOut}}
+          <p>
+            {{_ "residencies.note1"}}
+            <a
+              class="add-new-residency-link"
+              >{{_ "residencies.clickHere"}}</a
```

Comment:
Instead of the text "click here"  being a link. Lets create a small action button with the verb "End Residency". Then, `note1` should just be a descriptive sentence about the context, rather than combined with an action link.